### DISCUSSION TITLE
How to remove email field from JWT token ?

### DISCUSSION THREAD

**Discussion by geminigeek** [payloadcms/payload#613] [Q&A]
hi,

is there any way to remove email key from jwt tokens ?

it seems email is used to identify the user from db, by using id we can remove email from token, also we will be using default index by using id which will make auth query very fast!

**Comment by jmikrut** [comment]
Hey @geminigeek — currently you can't remove properties from JWT - only add them. PS - we currently _**do**_ index the `email` field so auth queries should indeed be quite fast even if we are querying not on ID. 

How come you'd want to _**remove**_ the email field?

**Reply by geminigeek** [reply]
hi jmikrut,

i am trying to implement an OTP / SMS authentication flow , now i am forced to create a dummy email till the user updates an actual email for the jwt flow to work , my suggestion is  "the user details queries should run with user Id saved in jwt rather than email" also i don't want to save email in a JWT,  i am not an expert but i don't like to save email in jwt.

Dee

**Comment by DanRibbens** [comment]
This is an old discussion so may not be relevant anymore, but this is now possible.

You can now set `disableLocalStrategy: true` inside of `auth` in your collections and it should no longer create the email field for you.
